pages/common/git-blame.md

```diff
@@ -0,0 +1,11 @@
+# git blame
+
+> Show what revision and author last modified each line of a file
+
+- Print file with author and revision per line
+
+`git blame {{file}}`
+
+- Print file with email and revision per line
```

Comment:
Perhaps this one could say "author email" and the one above "author name", for consistency and clarity.